### DISCUSSION TITLE
Add ability to create projects and configure FVM

### DISCUSSION THREAD

**Discussion by sameeerkashyap** [leoafarias/sidekick#136] [Ideas]
It would be great a new Flutter Project could be created with the option to select from fvm downloads and automatically initialized in the project. 

Also, future enhancement is the ability to install and modify options  `fvm`  CLI through sidekick, would make it an all in one awesome tool, I currently love using it, and these would be geat additions.

**Comment by aguilaair** [comment]
I'm turning this into a discussion, as it seems more appropriate

**Reply by aguilaair** [reply]
Also, seems to be very similar to #77

**Reply by sameeerkashyap** [reply]
@aguilaair yes, seems to be exactly the same, maybe this can be closed and continued on the same thread, I'm happy to help and make if the task can be divided into specific issued with specifications
